config/configgrpc/configgrpc_test.go

```diff
@@ -886,7 +889,8 @@ func TestClientInfoInterceptors(t *testing.T) {
 					require.NoError(t, tt.Shutdown(context.Background()))
 				}()
 
-				grpcClientConn, errClient := gcs.ToClientConn(context.Background(), componenttest.NewNopHost(), tt.TelemetrySettings)
+				var errClient error
+				grpcClientConn, errClient = gcs.ToClientConn(context.Background(), componenttest.NewNopHost(), tt.TelemetrySettings)
```

Comment:
you can use `defer func() {assert.NoError(t, grpcClientConn.Close())}()` instead.